cms/static/js/views/active_video_upload_list.js

```diff
@@ -169,6 +198,44 @@ define(
                 this.setStatus(data.cid, ActiveVideoUpload.STATUS_FAILED);
             },
 
+            hideErrorMessage: function() {
+                if (this.fileErrorMsg) {
+                    this.fileErrorMsg.hide();
+                    this.fileErrorMsg = null;
```

Comment:
Asset page page hides the notfication view when modal is closed (they have custom modal), and they make the errorMsg variable null when the process is done. 

We are doing the same but we have a different flow. We are hiding the notification view and making it null when file-chooser click event is called so that it won' come here again (Remember notification.shownAt property would be invalidated once hide is called, so next time, hide would render), that's why I am making this null and so does asset page at different locations.